cmd/kubeadm/app/cmd/join.go

```diff
@@ -152,6 +152,9 @@ func AddJoinConfigFlags(flagSet *flag.FlagSet, cfg *kubeadmapiext.NodeConfigurat
 	flagSet.StringVar(
 		&cfg.NodeName, "node-name", "",
 		"Specify the node name.")
+	flagSet.StringVar(
+		&cfg.NodeName, "cluster-name", "",
```

Comment:
We should have a default value here, right? Instead of current empty string, which will be misleading.

Change to `flagSet.StringVar(&cfg.NodeName, "cluster-name", cfg.NodeName, "Specify the cluster name.")`.

Reply:
Sure. Done